support-frontend/assets/helpers/page/page.ts

```diff
@@ -27,11 +27,16 @@ function getAbParticipations(): Participations {
 	const settings = getSettings();
 	const countryId: IsoCountry = Country.detect();
 	const countryGroupId: CountryGroupId = CountryGroup.detect();
+	const { selectedAmountsVariant } = getAmountsTestVariant(
+		countryId,
+		countryGroupId,
+		settings,
+	);
 	const abtestInitalizerData = {
 		countryId,
 		countryGroupId,
+		selectedAmountsVariant,
 	};
-
 	const participations: Participations = abTest.init(abtestInitalizerData);
 	const { amountsParticipation } = getAmountsTestVariant(
```

Comment:
We're now calling `getAmountsTestVariant` twice, you can remove the 2nd call, and just update the first to be:

```
const { amountsParticipation, selectedAmountsVariant } = getAmountsTestVariant(
		countryId,
		countryGroupId,
		settings,
	);
	```

Reply:
Ah yes, have neatened to this now, done